css/outline-color/demos/values/index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Варианты значений свойства — outline-color — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    html {
+      height: 100%;
+    }
+
+    body {
+      padding: 20px;
+      margin: 0;
+      min-height: 100%;
+      box-sizing: border-box;
+      background-color: #f7f7f7;
+      font-family: 'Roboto', sans-serif;
+    }
+
+    body,
+    .parent {
+      display: flex;
+      flex-wrap: wrap;
+      align-items: center;
+      justify-content: center;
+      grid-gap: 10px;
+    }
+
+    .parent {
+      margin: 20px;
+      padding: 20px;
+      outline: 2px solid;
+      outline-color: darkmagenta;
+    }
+
+    .outlined {
+      padding: 4px 10px;
+      outline: 2px solid;
+      outline-color: red;
+      font-size: 16px;
+      color: cornflowerblue;
+    }
+
+    .tomato {
+      outline-color: tomato;
+    }
+
+    .rgb {
+      outline-color: rgb(127, 155, 50);
+    }
+
+    .rgba {
+      outline-color: rgba(127, 155, 50, 0.25);
```

Comment:
```suggestion
      outline-color: rgb(127 155 50 / 0.25);
```

Мы стараемся использовать современный синтаксис RGB-цветов

Reply:
Done.